config/rbac/role.yaml

```diff
@@ -28,15 +28,61 @@ rules:
   verbs:
   - create
 
+# We need to manage Helm release daemonsets
+- apiGroups:
+  - falcon.crowdstrike.com
+  resources:
+  - daemonset
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+
+# We need to access the coreos nodes to protect them
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - privileged
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+
+# We need to easily pull images regardless of namespace
+- apiGroups:
+  - ""
+  - image.openshift.io
+  resources:
+  - imagestreamimages
+  - imagestreammappings
+  - imagestreams
+  - imagestreamtags
+  - imagetags
```

Comment:
Does this mean the helm chart and operator is openshift specific? Do we need to adjust for other distros?

Reply:
At this point, yes, since we need to run in a privileged SCC which is OCP specific. But as we go to other distros, we will look into addressing this.

Reply:
Should we remove the other k8s distros at https://github.com/CrowdStrike/falcon-helm#kubernetes-cluster-compatability ?

Or make a table about what is supported now vs tentative development roadmap?

| k8s | status |
| OCP | OCP 4.6+ |
| EKS | Planned for May 2020 |
| AKS | Planed for Jul 20202 |

etc

Reply:
> Should we remove the other k8s distros at https://github.com/CrowdStrike/falcon-helm#kubernetes-cluster-compatability ?

Those refer to the helm chart and not the helm operator.

> Or make a table about what is supported now vs tentative development roadmap?

Because we have both the helm chart and operator in the same repo and readme, I am going to rework the readme to bring more clarity.

Reply:
Spoke to @redhatrises rise. Future PR will clean up the REAMEs to make this more clear.